python/aie/compiler/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: MIT
```

Comment:
Everything else in aie is apache licensed and the recommendation we've had is to not change licenses.  I think not mixing is also a good idea, unless there is a particular reason.